src/Core/Console/Config.php

```diff
@@ -0,0 +1,137 @@
+<?php
+
+/*
+ * To change this license header, choose License Headers in Project Properties.
```

Comment:
Should this be changed?

Reply:
Totally, this is added by my IDE and has no business in the repo.